src/webgpu/web_platform/reftests/canvas_complex.html.ts

```diff
@@ -697,12 +697,18 @@ fn main(@builtin(global_invocation_id) GlobalInvocationID : vec3<u32>) {
       assert(ctx !== null, 'Failed to get WebGPU context from canvas');
 
       let usage = GPUTextureUsage.COPY_DST | GPUTextureUsage.RENDER_ATTACHMENT;
```

Comment:
should we add COPY_DST and RENDER_ATTACHMENT only when needed as well?

Reply:
It would be better, done!